schema/table.go

```diff
@@ -301,10 +292,6 @@ func (t *Table) ValidateName() error {
 	return nil
 }
 
-func (t *Table) ToArrowSchema() *arrow.Schema {
```

Comment:
We used this in quite a few places (e.g. the filetypes library), maybe we should keep it? We can mark it as deprecated instead.

I guess the idea here is to remove all the Arrow references from v1. If that's the way we want to go it's fine, we can update the couple of destinations that used it.